docs/book/src/tasks/experimental-features/runtime-sdk.md

```diff
@@ -0,0 +1,12 @@
+# Experimental Feature: Runtime SDK
+
+The Runtime SDK feature provides an extensibility mechanism that allows systems, products, and services built on top of Cluster API to hook into a workload cluster’s lifecycle.
```

Comment:
not sure how specific we want to be here, e.g. may be mention available Runtime Hook and particular Runtime Extensions

Reply:
I think this is definitely a work in progress just to get us started. My plan was to add documentation piece by piece as we add in some of the particular hooks, and hopefully add in some examples of extensions as they get developed and published.

Reply:
I think for now the current description is fine.

I think we should extend the documentation once the hooks are merged similar to how we did it with ClusterClass.

I expect us to have at least one additional page describing the cluster lifecycle hooks and one for topology mutation with user facing documentation.